share/spice/ron_swanson/ron_swanson.js

```diff
@@ -0,0 +1,31 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_ron_swanson = function(api_result){
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.error) {
+            return Spice.failed('ron_swanson');
+        }
+
+        // Render the response
+        Spice.add({
+            id: "ron_swanson",
+            name: "Ron Swanson Quote",
+            data: { 
+                title: function(){
+                    return '"' + api_result.quote + '" - Ron Swanson';
+                }()
+            },
```

Comment:
You can use the normalise function like this:
https://gist.github.com/MrChrisW/c03efafa711d61ef67ff